lib/husky_musher/app.py

```diff
@@ -152,3 +156,77 @@ def main():
 def set_cache_control(response):
     response.headers["Cache-Control"] = "no-store"
     return response
+
+
+def extract_user_info(environ: dict) -> Dict[str, str]:
+    """
+    Extracts attributes of the authenticated user, provided by UW's IdP via our
+    Shibboleth SP, from the request environment *environ*.
+
+    Keys of the returned dict match those used by our REDCap project.
+    """
+    return {
+        # We ask in REDCap for their @uw.edu email address, so provide it in
+        # that same format.  On ingest from REDCap into ID3C, we normalize it
+        # anyway.
+        "netid": f"{environ['uid']}@uw.edu",
```

Comment:
This has changed in REDCap to just the NetID.

Reply:
Good catch! Addressed in a new commit.